zebrad/src/components/sync.rs

```diff
@@ -387,19 +387,8 @@ where
                 self.update_metrics();
 
                 // If we have too many pending tasks, wait for some to finish.
-                //
-                // Starting to wait is interesting, but logging each wait can be
-                // very verbose.
-                if self.downloads.in_flight() > self.lookahead_limit {
-                    tracing::info!(
-                        tips.len = self.prospective_tips.len(),
-                        in_flight = self.downloads.in_flight(),
-                        lookahead_limit = self.lookahead_limit,
-                        "waiting for pending blocks",
```

Comment:
This log is really helpful for diagnosing sync issues. I'd like to keep it at info level.